crates/turborepo-lib/src/commands/unlink.rs

```diff
@@ -13,3 +25,67 @@ pub fn unlink(base: &mut CommandBase) -> Result<()> {
 
     Ok(())
 }
+
+fn unlink_spaces(base: &mut CommandBase) -> Result<()> {
+    let result =
+        remove_spaces_from_turbo_json(base).context("could not unlink. Something went wrong")?;
+
+    match result {
+        UnlinkSpacesResult::Unlinked => {
+            println!("{}", base.ui.apply(GREY.apply_to("> Unlinked Spaces")));
+        }
+        UnlinkSpacesResult::NoSpacesFound => {
+            println!(
+                "{}",
+                base.ui.apply(GREY.apply_to("> No Spaces config found"))
+            );
+        }
+    }
+
+    Ok(())
+}
+
+pub fn unlink(base: &mut CommandBase, target: Option<LinkTarget>) -> Result<()> {
+    let target = target.unwrap_or(LinkTarget::RemoteCache);
+
+    match target {
+        LinkTarget::RemoteCache => {
+            unlink_remote_caching(base)?;
+        }
+        LinkTarget::Spaces => {
+            unlink_spaces(base)?;
+        }
+    }
+    Ok(())
+}
+
+fn remove_spaces_from_turbo_json(base: &CommandBase) -> Result<UnlinkSpacesResult> {
+    let turbo_json_path = base.repo_root.join("turbo.json");
+
+    if !turbo_json_path.exists() {
+        return Err(anyhow!("turbo.json not found."));
+    }
+
+    let turbo_json_file = File::open(&turbo_json_path)?;
+    let mut turbo_json: TurboJson = serde_json::from_reader(turbo_json_file)?;
+    let has_spaces_id = turbo_json
+        .experimental_spaces
+        .unwrap_or(SpacesJson {
```

Comment:
We could derive `Default` on `SpacesJson` and then do `unwrap_or_default`

Reply:
Love it